### PR TITLE
CI script for AWS CodeBuild.

### PR DESCRIPTION
**Pull request by ShaleXIONG**
CI script for AWS CodeBuild:
- build and run all tests on Trustzone backend.
- build and run Durango tests on SGX backend.

### CODE REVIEW

**Comment by dominic-mulligan-arm** [comment]
@ShaleXIONG is this still a draft?  I think you said this is not ready for merge yet, right?

**Comment by ShaleXIONG** [comment]
I am waiting for the CI result. I believe it should be good. Once it passes I will raise an official merge request.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Pull request merged**
